src/components/common/PeopleAndRoles/ListPeopleAndRoles.vue

```diff
@@ -303,72 +316,51 @@ export default class ListPeopleAndRoles extends Mixins(CommonMixin) {
   @Prop() readonly currentCompletingParty: OrgPersonIF
 
   @Prop({ default: false })
-  readonly isSummaryView!: boolean
+  readonly isSummaryView: boolean
 
+  /** Whether to perform validation. */
   @Prop({ default: false })
-  readonly validate!: boolean
+  readonly validate: boolean
 
-  /** Whether we have all required parties. */
+  /** Whether OrgPersons list is valid. */
   @Prop({ default: true })
-  readonly haveRequiredParties: boolean
+  readonly validOrgPersons: boolean
+
+  // Store getter
+  @Getter getPeopleAndRoles!: OrgPersonIF[]
 
   // declaration for template
   readonly isSame = isSame
 
-  /** The name section validity state (when prompted by app). */
-  get invalidOrgPersons (): boolean {
-    return (this.validate && this.renderOrgPersonForm) || !this.haveRequiredParties
+  /** This component's validity state (for error styling). */
+  get isComponentInvalid (): boolean {
+    // if prompted by app or if this is a single-page correction filing
+    // and
+    // if form is open or if list is invalid
+    return (this.validate || this.isCorrectionFiling) && (this.renderOrgPersonForm || !this.validOrgPersons)
   }
 
   /** The current orgPersons list. */
   get currentPeopleAndRoles (): Array<OrgPersonIF> {
-    return this.isSummaryView ? this.filterRemovedOrgPersons(this.peopleAndRoles) : this.peopleAndRoles
-  }
-
-  /** Helper method to filter REMOVED orgPersons from PeopleAndRoles list. */
-  private filterRemovedOrgPersons (peopleAndRoles: Array<OrgPersonIF>): Array<OrgPersonIF> {
-    return peopleAndRoles.filter(orgPerson => !orgPerson.actions?.includes(ActionTypes.REMOVED))
+    if (this.isSummaryView) {
+      // return list without REMOVED org-persons
+      return this.getPeopleAndRoles.filter(orgPerson => !orgPerson.actions?.includes(ActionTypes.REMOVED))
+    }
+    return this.getPeopleAndRoles
   }
 
   /** Headers for the person table. */
   readonly tableHeaders = ['Name', 'Mailing Address', 'Delivery Address', '']
 
   /** V-model for dropdown menus. */
-  private dropdown: Array<boolean> = []
-
-  /**
```

Comment:
I moved a bunch of helper methods from here to OrgPersonMixin so they can be centralized/reused.